### PR TITLE
Sending the actual error as a string and not the event

### PR DESCRIPTION
**Pull request by pasviegas**

### CODE REVIEW

File: src/com/rollbar/notifier/RollbarNotifier.as [hunk 1]
```diff
@@ -131,8 +131,8 @@ package com.rollbar.notifier {
             return handleError(newError);
         }
 
-        public function handleOtherEvent(event:*):String {
-            var newError:Error = new Error("A non-Error or ErrorEvent was thrown and not caught: " + event.toString());
+        public function handleOtherEvent(error:*):String {
+            var newError:Error = new Error("A non-Error or ErrorEvent was thrown and not caught: " + error.toString());
```
Comment (brianr) [inline src/com/rollbar/notifier/RollbarNotifier.as]: @pasviegas Would it be more useful to send both the event and the error? e.g. include both `event.toString()` and `event.error.toString()`

Comment (pasviegas) [inline src/com/rollbar/notifier/RollbarNotifier.as]: Event will always be "[UncaughtErrorEvent type="uncaughtError" bubbles=true cancelable=true eventPhase=2]" so I don't see why sending it really. :)

**Comment by rivkahstandig3636** [comment]
@brianr Should we merge this or close it?

**Comment by rivkahstandig3636** [comment]
Closing due to inactivity.

**Pull request closed without merging**